babel/board.js

```diff
@@ -88,9 +88,48 @@ export class Board {
     get availableTypes() {
         return _.uniq(_.flatten(this.orbs)).sort();
     }
+    
+    /** 
+      * 1. logs data for each match and replaces each orb with 'X'
+      * 2. replaces each 'X' and all above orbs with either the orb directly above or a random new orb
+      * 3. returns the match data -> [[match1Type, match1Amount], [match2Type, match2Amount], ...]
+      */
```

Comment:
Bookmark this site: http://usejsdoc.org/

We'll go over how to properly document javascript functions later. For now, this is fine.